pkg/kubelet/container/runtime.go

```diff
@@ -52,6 +52,10 @@ type Runtime interface {
 	// TODO(yifan): Pull/Remove images
 }
 
+type ContainerRunner interface {
```

Comment:
Why not have this be part of Runtime above? I worry about having an interface per function call.